lib/matplotlib/backends/qt_editor/figureoptions.py

```diff
@@ -101,9 +105,8 @@ def prepare_data(d, init):
                 sorted(short2name.items(),
                        key=lambda short_and_name: short_and_name[1]))
 
-    curvelabels = sorted(linedict, key=cmp_key)
-    for label in curvelabels:
-        line = linedict[label]
+    sorted_labels_and_curves = sorted(labeled_lines, key=cmp_key)
+    for label, line in sorted_labels_and_curves:
```

Comment:
```suggestion
    for label, line in sorted(labeled_lines, key=cmp_key):
```
Can now be inlined because it's not needed anywhere else.